data-pipeline/src/pipeline/pre_processing.py

```diff
@@ -434,9 +419,7 @@ def prepare_aar_data(aar_path, current_year: int):
     aar = aar[~aar["URN"].isna()]
     aar = aar[~(aar["ACADEMYTRUSTSTATUS"] == "1 day")]
 
-    if (current_year < 2023) or (
```

Comment:
What is the history behind this, and the above? Columns mapped incorrectly for some data sets but now reverted?

Reply:
The AAR mappings had just got mixed up in the spreadsheet and have only just been spotted.